docs/Zendesk-batchsource.md

```diff
@@ -77,8 +79,17 @@ corresponding CDAP types.
 | Array                  | Array     |
 | Record                 | Record    |
 
-Data Duplication issues with Zendesk APIs
+
+Limitations
 ----------
+
+Zendesk supports three types of pagination: [offset](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-offset-pagination/), [time-based](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-cursor-pagination/) and [cursor](https://developer.zendesk.com/documentation/developer-tools/pagination/paginating-through-lists-using-cursor-pagination/) pagination. 
+The following issues occur when the Zendesk API uses these pagination.
+- Offset and Time-Based Pagination: Data Duplicacy
+- Cursor Pagination: Data Loss
+
+### Data Duplication issues with Zendesk APIs
+
 This plugin can return duplicate records for time-based exports as mentioned in Zendesk docs.
```

Comment:
We need to update this section to match the changes in Limitations section: offset also applies: https://docs.google.com/document/d/1cNQqpMcEQXfN2aBDpVYCiBGaTyfh8W4WAkQXoOVgvdY/edit#heading=h.gjdgxs

Reply:
done